### DISCUSSION TITLE
Is a PR to a PR counted as a contribution to the main project?

### DISCUSSION THREAD

**Discussion by neurorepro** [community/community#22496] [Pull Requests]
<p>I am interested in project X, and i saw that  was working on a PR Y to project X. Since the PR needed more work, i forked repo Y to add content to 's PR.  merged my PR in his PR Y, so now the content i contributed to is part of the PR Y to X.</p>
<p>Will my contribution be counted towards project X (number of lines, project X contributor, etc.) ? Or what could i ask to be done so that my contribution is ackowledged in the main repo X ?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>As long as the author information in your commits is correct and retained (no squash merge!), those commits will be recognized as yours. :slightly_smiling_face:</p>

**Comment by neurorepro** [comment]
<p>The text <code>&lt;other_user&gt;</code> was removed from my post, so it became very hard to understand. The original text i wrote was like this:</p>
<p>=============<br>
I am interested in project X, and i saw that <code>&lt;other_user&gt;</code> was working on a PR Y to project X. Since the PR needed more work, i forked repo Y to add content to <code>&lt;other_user&gt;</code>'s PR. merged my PR in his PR Y, so now the content i contributed to is part of the PR Y to X.</p>
<p>Will my contribution be counted towards project X (number of lines, project X contributor, etc.) ? Or what could i ask to be done so that my contribution is ackowledged in the main repo X ?</p>
